corehq/apps/export/export.py

```diff
@@ -465,25 +465,29 @@ def _get_base_query(export_instance):
         )
 
 
-def rebuild_export(export_instance, last_access_cutoff=None, filters=None):
+def rebuild_export(export_instance, filters=None):
     """
     Rebuild the given daily saved ExportInstance
     """
-    if _should_not_rebuild_export(export_instance, last_access_cutoff):
-        return
     filters = filters or export_instance.get_filters()
     export_file = get_export_file([export_instance], filters or [])
     with export_file as payload:
         save_export_payload(export_instance, payload)
 
 
-def _should_not_rebuild_export(export, last_access_cutoff):
+def _should_rebuild_export(export, last_access_cutoff=None):
```

Comment:
nit: should remove leading `_` since this is now imported from other modules (no longer private).

Reply:
curious why the default was added (`last_access_cutoff=None`) since this function is never called with just a single argument?

Reply:
Not sure, will remove.